deployment/updates/scripts/2021_07_26_deploy_zoom_content_conversion_data.php

```diff
@@ -0,0 +1,16 @@
+<?php
```

Comment:
Remove all installations other than static content.
Once we enable the feature cross-partner we will add these deployment scripts